### PR TITLE
fixes #9343 - Actkey copies auto-attach preference, BZ1173724

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

File: app/controllers/katello/api/v2/activation_keys_controller.rb [hunk 1]
```diff
@@ -99,7 +99,8 @@ def copy
       @new_activation_key.reload
       @new_activation_key.update_attributes!(
                                              :service_level => @activation_key.service_level,
-                                             :release_version => @activation_key.release_version
+                                             :release_version => @activation_key.release_version,
+                                             :auto_attach => @activation_key.auto_attach
```
Comment (jlsherrill) [inline app/controllers/katello/api/v2/activation_keys_controller.rb]: i feel like i should know this, but there is there a reason to not set all 3 of these attributes in ActivaitonKey#copy  ?

Comment (jlsherrill) [inline app/controllers/katello/api/v2/activation_keys_controller.rb]: ah, i see why, we should probably dynflow this entire process at some point, can you file an issue for that?

**Comment by jlsherrill** [comment]
ACK pending issue filed

**Comment by cfouant** [comment]
http://projects.theforeman.org/issues/9432 created

**Pull request merged**